src/WorkflowServiceProvider.php

```diff
@@ -53,6 +53,17 @@ protected function configPath()
     {
         return __DIR__ . '/../config';
     }
+    
+    protected function publishPath($configFile)
+    {
+        if (function_exists('config_path')) {
```

Comment:
No need for the intermediate variable here, can just return `config_path(...` if the function exists or `base_path(...` otherwise

Reply:
Looks good, otherwise!

Reply:
Thanks. The code has been modified.